spec/rails_helper.rb

```diff
@@ -36,4 +36,4 @@
 
 # only look for driver updates if the previous check was
 # longer ago than a specified number of seconds
-Webdrivers.cache_time = 86_400
+# Webdrivers.cache_time = 86_400
```

Comment:
@ovasdi - should we just delete this? Wasn't sure if we just wanted to comment while working locally.

Reply:
Let's remove it seeing as [Webdrivers was removed](https://github.com/artsy/rosalind/pull/739/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fL412).

Reply:
Done in [5c9ae9c](https://github.com/artsy/rosalind/pull/739/commits/5c9ae9cb9107e9e449d54622e44f2f355fdc7e57)